20_kata.py

```diff
@@ -0,0 +1,15 @@
+# https://www.codewars.com/kata/55c6126177c9441a570000cc
+
+import re
+
+
+def order_weight(strng):
+
+    cleaned_string = re.findall(r"\d+", strng)
+    cleaned_string = sorted(cleaned_string)
+
+    def sum_of_digit(digits):
```

Comment:
Good variable naming for function's argument! The function name has a typo, should be named `sum_of_digits` with the `s` at the end indicating it's plural.